platform-server-kernel/src/main/java/uno/cod/platform/server/core/dto/challenge/ChallengeCreateDto.java

```diff
@@ -10,6 +10,9 @@
     private String name;
 
     @NotNull
+    private Long organizationId;
+
```

Comment:
do we need the public flag here as well? or do we only set a challenge public after creation?